test/promql/tests/data_plane/notready_pods_test.yaml

```
@@ -199,6 +199,12 @@ tests:
       - series: 'kube_pod_status_unschedulable{namespace="build-templates-e2e", pod="test-pod", source_cluster="cluster01"}'
         values: '0x29'
 
+      # Pod is in the Failed state and scheduled, but it's in the 'konflux-ci' namespace so it's ignored.
+      - series: 'kube_pod_status_phase{pod="test-pod", namespace="konflux-ci", phase="Failed"}'
+        values: '1x14'
+      - series: 'kube_pod_status_unschedulable{namespace="konflux-ci", pod="test-pod", source_cluster="cluster01"}'
+        values: '0x14'
```

This won't fire even if the namespace isn't `konflux-ci` because there aren't enough values to match the `interval` and `for` fields on the alerting rule. See all the time series above.

```suggestion
        values: '1x29'
      - series: 'kube_pod_status_unschedulable{namespace="konflux-ci", pod="test-pod", source_cluster="cluster01"}'
        values: '0x29'
```

---

Updated